### PR TITLE
Added a sample code to show auto punctuation

### PR DESCRIPTION
**Pull request by happyhuman**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by happyhuman** [comment]
@beccca , @alexander-fenster , @ace-n can you please review this PR?

File: samples/recognize.v1p1beta1.js [hunk 1]
```diff
@@ -132,6 +132,58 @@ function syncRecognizeModelSelectionGCS(
...
+  // const sampleRateHertz = 16000;
+  // const languageCode = 'BCP-47 language code, e.g. en-US';
+
+  const config = {
+    encoding: encoding,
+    languageCode: languageCode,
+    enableAutomaticPunctuation: true,
+  };
+  const audio = {
+    content: fs.readFileSync(filename).toString('base64'),
```
Comment (beccca) [inline samples/recognize.v1p1beta1.js]: Out of curiosity, does it also work if you do this?

```js
const audio = {
  content: {
    filename: filename // <--- the nodejs libraries sometimes check for the `filename` key and
  }                    // will do the readFileSync/base64 for the user (may or may not work here)
};
```

If so, cool, use that. If not, also cool.

File: samples/system-test/recognize.v1p1beta1.test.js [hunk 2]
```diff
@@ -25,13 +25,16 @@ const {runAsync} = require(`@google-cloud/nodejs-repo-tools`);
 const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
 const cmd = `node recognize.v1p1beta1.js`;
 const cwd = path.join(__dirname, `..`);
-const filename = `Google_Gnome.wav`;
-const filepath = path.join(__dirname, `../resources/${filename}`);
-const text = `the weather outside is sunny`;
+const filename1 = `Google_Gnome.wav`;
+const filename2 = `commercial_mono.wav`;
```
Comment (beccca) [inline samples/system-test/recognize.v1p1beta1.test.js]: I can't find this being used anywhere? Why was this variable added to the tests?

If `filename2` and `filepath2` and `test2` aren't used, the changes here should be reverted back to the original `filename`, `filepath`, and `text` vars

Comment (beccca) [reply samples/system-test/recognize.v1p1beta1.test.js]: Nevermind, found it, missed it the first time

File: samples/recognize.v1p1beta1.js (hunk 1 above)
Comment (happyhuman) [reply samples/recognize.v1p1beta1.js]: Just tried it, it did not work (although I wish it did).    :)

**Pull request merged**